javascript.js

```diff
@@ -0,0 +1,20 @@
+// Set variavles
+var user = []; //inline coment
+
+/**
```

Comment:
Typo inside comment in token 'togeter'. Possible corrections:together (43),toaster (0)